internal/backend/rclone/backend.go

```diff
@@ -222,7 +222,7 @@ func newBackend(cfg Config, lim limiter.Limiter) (*Backend, error) {
 	// send an HTTP request to the base URL, see if the server is there
 	client := &http.Client{
 		Transport: debug.RoundTripper(tr),
-		Timeout:   60 * time.Second,
+		Timeout:   time.Duration(cfg.Timeout) * 60 * time.Second,
```

Comment:
just use a plain `cfg.Timeout` here. At the moment we have square seconds here.

Reply:
Oh yea, good catch